adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -80,6 +81,13 @@
     // Example of key after composition: fare_idroute_idorigin_iddestination_idcontains_id
     private final Map<String, FareRule> fareRuleCollection = new HashMap<>();
 
+    // Map containing Frequency entities. Entities are mapped on a composite key made of the values found in the
+    // columns of GTFS file frequencies.txt:
+    // - trip_id
+    // - start_time
+    // Example of key after composition: trip_idstart_time
+    private final Map<String, Frequency> frequencyCollection = new HashMap<>();
```

Comment:
For legibility and consistency with most of the declarations here, `frequencyCollection` could be renamed as `frequencyPerTripIdStartTime`